docs/getting_started/switching_composer.rst

```diff
@@ -6,17 +6,20 @@ How to switch to Composer
 .. vale on
 
 Until Mautic 4, you could download Mautic as a ZIP file and install it on any PHP server. 
-However, many Users were running into installation and update errors, many of which caused considerable frustration and in some cases, significant business disruption. Next to that, Mautic recently introduced the :ref:`Mautic Marketplace` which isn't compatible with this installation method.
+However, many Users were running into installation and update errors, many of which caused considerable frustration and in some cases, significant business disruption. 
```

Comment:
```suggestion
However, many Mautic users were running into installation and update errors, many of which caused considerable frustration and in some cases, significant business disruption. 
```

Here we are talking about the folks who use Mautic rather than Users in the Mautic user management, so lower case U.